server/services/communityApiService.ts

```diff
@@ -30,4 +37,10 @@ export default class CommunityApiService {
     logger.info(`getting user details for username ${username}`)
     return (await this.restClient(token).get({ path: `/secure/users/${username}/details` })) as DeliusUser
   }
+
+  async getOffenderByCRN(crn: string): Promise<DeliusOffender> {
```

Comment:
again, can we make this `getServiceUserByCRN`? nDelius may refer to people as `offender`s, so I can see why you used this name, but it'd be good to use the new terminology in our service.